torch/_inductor/runtime/triton_helpers.py

```diff
@@ -627,3 +631,42 @@ def x_grid_barrier(sem):
 
     # TODO(jansel): is this needed?
     tl.debug_barrier()
+
+
+def triton_builtin(f: _T) -> _T:
```

Comment:
You can bound the TypeVar to some class / subclass here if you want to remove the attr-defined error

Reply:
IMO not worth it since no code in this repository uses this attribute.